ratatui/benches/main/constraints.rs

```diff
@@ -0,0 +1,59 @@
+use std::hint::black_box;
+use std::rc::Rc;
+
+use criterion::{Criterion, criterion_group};
+use ratatui::layout::Constraint::{Fill, Length, Max, Min, Percentage, Ratio};
+use ratatui::layout::{Direction, Layout, Rect};
+
+const SPLIT_BY: u16 = 10;
+
+fn generate_layout(constraint: &str, area: Rect) -> Rc<[Rect]> {
```

Comment:
As a general rule in Ratatui, we write functions in calling order (outside-in) to make it easier to read code top to bottom. Can you please move this below the other function.